### ISSUE TITLE
Macro expansion referencing the core crate is not hygienic

### ISSUE THREAD

**Issue by cswinter** [shepmaster/snafu#235]
If you declare a module with the name `core` in a file that uses `Snafu` derive macro, compilation fails with a bunch of errors like the following:

```
error[E0433]: failed to resolve: could not find `convert` in `core`
  --> src/lib.rs:40:17
   |
40 | #[derive(Debug, Snafu)]
   |                 ^^^^^ could not find `convert` in `core`
   |
   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by shepmaster** [comment]
This _should_ be an easy fix; I believe that the macro needs to output `::core` instead of `core`. We might as well update it to `::snafu` as well.

**Comment by shepmaster** [comment]
Fixed in 0.6.8
